README.md

```diff
@@ -71,6 +71,7 @@ Installation instructions for previous releases can be found in [Metrics Server
 
 Metrics Server | Metrics API group/version | Supported Kubernetes version
 ---------------|---------------------------|-----------------------------
+0.7.x          | `metrics.k8s.io/v1beta1`  | 1.28+
```

Comment:
In fact, the e2e test now covers versions 1.26, 1.27 and 1.28, and metrics-server can all run normally.
Why is it set to 1.28+ here?

Reply:
we should stick to 1.19+, there was no breaking changes since then